docs/install/Knative-custom-install.md

```diff
@@ -71,24 +71,40 @@ with Knative.
 The following Knative installation files are available:
 
 - **Serving Component and Observability Plugins**:
-  - https://github.com/knative/serving/releases/download/{{< version >}}/serving.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-logs-elasticsearch.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-metrics-prometheus.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-jaeger.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-jaeger-in-mem.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-zipkin.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-zipkin-in-mem.yaml
+  - https://github.com/knative/serving/releases/download/{{<
```

Comment:
These are all broken too - the shortcode must remain on a single line 
`{{< version >}}`

Reply:
Is there any way for it to not have spaces?

Reply:
no not for shortcodes in Hugo (i tried w/o spaces but they result in printing directly to the ouput - hugo wont process it)

Reply:
Related FYI: https://github.com/knative/docs/issues/1678